_posts/blog/2016-11-30-tcp-and-bbr.md

```diff
@@ -0,0 +1,26 @@
+
+## How Does BBR Make the Internet Work Better?
+
+BBR prevents traffic congestion by measuring how fast the network can actually deliver data, and then using that rate to construct a model of the network's capabilities. This model is then used to determine how much data can be sent at a time. By using this model (and repeatedly re-measuring to verify that the model remains correct) BBR-based TCP can transmit data at the right rate without causing congestion and loss of in-flight data. Even more, this model indicates that there is a lot of spare network capacity that older implementations of TCP are unable to utilize. By discovering this available capacity and making it available for use, BBR unlocks network capabilities that have always existed, but have never before been easily used.  Using BBR TCP means that the right rate of traffic transmission is much more quickly discovered and that less traffic has to be re-sent. Additionally, the BBR TCP rate is frequently faster than the rate discovered by older TCP implementations. This makes for a faster everything -- measurement, connection, etc., and increases accuracy of measurement.  Additionally, this update can be implemented on just the server side, and so doesn’t require patches for individual browsers or smartphones. The move to BBR-based TCP will start to happen without most of us even noticing.
```

Comment:
The move to BBR-based TCP will start -> The move to BBR-based TCP has already started